neon-animation-behavior.html

```diff
@@ -37,9 +37,10 @@
 
     },
 
-    registered: function() {
-      new Polymer.IronMeta({type: 'animation', key: this.is, value: this.constructor});
```

Comment:
This is a breaking change. Whether we like it or not, `<iron-meta>` is a public accessor into a global map of whatever goes into it.